<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MI_MA</class>
 <widget class="QWidget" name="MI_MA">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MI_MA</string>
  </property>
  <widget class="QLineEdit" name="mima">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>70</y>
     <width>110</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>你还有 3 次输入密码的机会~</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>41</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>密 码：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>70</y>
     <width>51</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/Kao_he/shen.png);</string>
   </property>
   <property name="text">
    <string>确 认</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>300</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/Kao_he/qian.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>label_13</zorder>
  <zorder>mima</zorder>
  <zorder>label_11</zorder>
  <zorder>label_12</zorder>
  <zorder>pushButton</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MI_MA</receiver>
   <slot>ti_jiao()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>231</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>213</x>
     <y>-4</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>ti_jiao()</slot>
 </slots>
</ui>
